addons/event/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def fill_event_mail_notification_type(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE event_mail
+        SET notification_type = 'mail'
+        WHERE notification_type IS NULL
+        """
+    )
+    openupgrade.logged_query(
+        cr, """
+        UPDATE event_type_mail
+        SET notification_type = 'mail'
+        WHERE notification_type IS NULL
+        """
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    fill_event_mail_notification_type(env.cr)
+    openupgrade.load_data(env.cr, 'event', 'migrations/13.0.1.0/noupdate_changes.xml')
```

Comment:
you should delete the record translations of event_reminder and event_subscription templates.